modules/build/src/main/scala/scala/build/internal/util/WarningMessages.scala

```diff
@@ -123,4 +123,11 @@ object WarningMessages {
 
   val mainScriptNameClashesWithAppWrapper =
     "Script file named 'main.sc' detected, keep in mind that accessing it from other scripts is impossible due to a clash of `main` symbols"
+
+  def deprecatedWarning(old: String, `new`: String) =
+    s"Using `$old` is deprecated, use `${`new`}` instead"
```

Comment:
NIT use `'` instead of ` in command line logs
```suggestion
    s"Using '$old' is deprecated, use '${`new`}' instead"
```